scripts/qtsupport.py

```diff
@@ -99,7 +100,7 @@
     "native": "NOT currently supported, except for PayJoin (command line only)",
     "console_log_level": "one of INFO, DEBUG, WARN, ERROR; INFO is least noisy;\n" +
     "consider switching to DEBUG in case of problems.",
-    "absurd_fee_per_kb": "maximum satoshis/kilobyte you are willing to pay,\n" +
+    "absurd_fee_per_kb": "maximum amount per kilobyte you are willing to pay,\n" +
```

Comment:
Why this change? I don't get it.

Reply:
Because you can now enter either BTC or sats there.

Reply:
I see. Hadn't noticed that it was added there. Indeed it's a nice addition, unfortunate that `tx_fees` being a more complex setting cannot be done the same way. Still, good.